### PR TITLE
bin/crew: try relative symlink

### LINKED ISSUE #5603: crew symlink broken
image

### PR DESCRIPTION
**Pull request by saltedcoffii**
another potential solution to fix #5603

### CODE REVIEW

**Comment by supechicken** [comment]
Does this require user to run `/usr/local/lib/crew/bin/crew` after `crew update` to get new symlink?

File: bin/crew [hunk 1]
```diff
@@ -454,7 +454,7 @@ def update
   end
 
   # Update the symlink
-  FileUtils.symlink "#{CREW_LIB_PATH}/bin/crew", "#{CREW_PREFIX}/bin/crew", force: true, verbose: @fileutils_verbose
+  FileUtils.symlink "../lib/crew/bin/crew", "#{CREW_PREFIX}/bin/crew", force: true, verbose: @fileutils_verbose
```
Comment (uberhacker) [inline bin/crew:457]: This is worth a shot.  Bump crew version in `lib/const.rb`?

**Comment by uberhacker** [comment]
> Does this require user to run `/usr/local/lib/crew/bin/crew` after `crew update` to get new symlink?

You might be on to something here @supechicken.

File: crew [hunk 2]
```diff
@@ -0,0 +1 @@
+bin/crew
```
Comment (uberhacker) [inline crew:1]: I don't think this will execute like you might expect.  Better to make it a symlink.

Comment (saltedcoffii) [reply crew:1]: it is a symlink

Comment (saltedcoffii) [reply crew:1]: github is just displaying it weirdly

File: bin/crew [hunk 3]
```diff
@@ -454,7 +454,7 @@ def update
   end
 
   # Update the symlink
-  FileUtils.symlink "#{CREW_LIB_PATH}/bin/crew", "#{CREW_PREFIX}/bin/crew", force: true, verbose: @fileutils_verbose
+  #FileUtils.symlink "../lib/crew/bin/crew", "#{CREW_PREFIX}/bin/crew", force: true, verbose: @fileutils_verbose
```
Comment (uberhacker) [inline bin/crew:457]: Can we just remove this altogether and bump the crew version again?

**Pull request merged**